docs/installation.md

```diff
@@ -40,3 +40,31 @@ Valkyrie also uses git quite extensively. So if you don't already have it
 installed, go ahead and do that next.
 
  * http://git-scm.com/downloads
+
+### Notes for Linux 
+**NFS server installed on the host is a requirement for valkyrie**
+* On Debian/Ubuntu based distributions, install the nfs-kernel-server package.
+* On RedHat based distributions, here is a [good guide](http://computernetworkingnotes.com/network-administration/how-to-configure-nfs-server-in-rhel-6.html) for installing and configuring the nfs server.
+
+**Authorize NFS connections through your host's firewall to the VM**
```

Comment:
Everything below this point should probably go in a separate 'Troubleshooting' page.